Analysis/src/SampleDescriptor.cpp

```diff
@@ -28,6 +28,20 @@ void AnalyzerSetup::CreateLimitSetups()
         }
     }
 }
+void AnalyzerSetup::ConvertToEventRegion(const std::vector<std::string>& regions, EventRegionSet& event_region_set)
+{
+    std::map<std::string, EventRegion> predefined_regions = {
```

Comment:
Don't define any map, use `analysis::Parse` instead that uses the I/O operator (`>>`) to insure the consistent string representation in the code.